src/lib/audit-log.ts

```
@@ -0,0 +1,31 @@
+
+export type AuditAction =
+    | 'INDUSTRY_LIST'
+    | 'INDUSTRY_LIST_ERROR'
+    | 'INDUSTRY_CREATE'
+    | 'INDUSTRY_CREATE_ERROR'
+    | 'ORGANIZATION_LIST'
+    | 'ORGANIZATION_LIST_ERROR'
+    | 'ORGANIZATION_CREATE'
+    | 'ORGANIZATION_CREATE_ERROR';
+
+export async function auditLog(action: AuditAction, data?: Record<string, any>) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)